### PR TITLE
Prune images in Cloud Sites articles 160616-1

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-271adea316/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-358f9e3957/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bc4ad42e44/how-to/)

File: content/cloud-sites/dns-management-how-to-update-your-ip-address.md [hunk 1]
```diff
@@ -5,47 +5,40 @@ title: 'Update your DNS IP address'
...
 3.  Select the domain name for which you want to obtain DNS information.
 
 4.  Click on the **DNS** tab toward the top of your screen.
 
-     <img src="{% asset_path cloud-sites/dns-management-how-to-update-your-ip-address/sitesdnsbar.png %}" alt="" />
-
-5.  Scroll down to the **DNS Management** section within the DNS tab. This area shows all 
-    of the DNS entries for. The two entries that you need are the www.domain.com, and the 
+5.  Scroll down to the **DNS Management** section within the DNS tab. The two entries that you need are the www.domain.com, and the
     root domain.com as indicated in the following screenshot.
```
Comment (catlook) [inline content/cloud-sites/dns-management-how-to-update-your-ip-address.md]: you deleted the screenshot, so remove the "following screenshot" line.

File: content/cloud-sites/dns-management-how-to-update-your-ip-address.md [hunk 2]
```diff
@@ -55,24 +48,16 @@ You can also edit and update your DNS records through the DNS page of the Cloud
...
-
 3.  Click on the domain name for which you want to obtain the DNS information.
 
-     <img src="{% asset_path cloud-sites/dns-management-how-to-update-your-ip-address/1216-2_0.png %}" alt="" />
-
-4.  On the **Domain Details** page, the DNS entries are located under the **Records** heading. 
-    The two entries that you need are the www.example-domain.com and the root example-domain.com, 
+4.  On the **Domain Details** page, the DNS entries are located under the **Records** heading.
+    The two entries that you need are the www.example-domain.com and the root example-domain.com,
     as indicated in the following screenshot.
```
Comment (catlook) [inline content/cloud-sites/dns-management-how-to-update-your-ip-address.md]: you deleted the screenshot, so remove the "following screenshot" line.

File: content/cloud-sites/getting-started-with-cloud-sites-accessing-billing-information.md [hunk 3]
```diff
@@ -50,8 +50,6 @@ In the Cloud Sites Control Panel, you can access usage data by clicking
 The **Reports** section opens to the **Usage Data** tab, as shown in the
 following figure.
```
Comment (catlook) [inline content/cloud-sites/getting-started-with-cloud-sites-accessing-billing-information.md]: you deleted the figure so remove the "following figure" line.

**Comment by catlook** [comment]
Ok Kyle, just fix a couple of things in  content/cloud-sites/dns-management-how-to-update-your-ip-address.md  (comments inline), and we can merge.

**Pull request closed without merging**